pombola/core/management/commands/core_sync_everypolitician_uuid.py

```diff
@@ -0,0 +1,54 @@
+from everypolitician import EveryPolitician
+
+from django.core.management.base import BaseCommand
+
+from pombola.core.models import Person
+
+
+class Command(BaseCommand):
+    help = 'Sync EveryPolitician UUID to Person.everypolitician_uuid field'
```

Comment:
"Person.everypolitician_uuid field" - I suspect this is an old comment?

Reply:
:+1: